lib/src/group_changes.dart

```diff
@@ -32,12 +32,36 @@ class MinMax {
 /// sets as keys in a later map, so we use the keys, and can recover the sets.
 Map<String, List<String>> configSets;
 
+class Result {
+  Result(Map<String, dynamic> data)
+      : previous = data['previous_result'],
+        current = data['result'],
+        expected = data['expected'],
+        // Different methods of accessing BigQuery give different types here.
```

Comment:
Should/can this be handled at a lower level?

Reply:
No action taken. One of the two methods should be standardized on, and this may already have happened.